configuration/locations/locations.csv

```diff
@@ -41,9 +36,5 @@ bb0e512e-d225-11e4-9c67-080027b662ec,TRUE,Labour Ward,,,,,,,,,,,,,,,
 bb0eb071-d225-11e4-9c67-080027b662ec,TRUE,Labour Ward - 1,,,,,,,,,,,,,,,
 e48fb2b3-d490-11e5-b193-0800270d80ce,TRUE,General Ward - Room 2,,,,,,,,,,,,,,,
 8d6c993e-c2cc-11de-8d13-0010c6dffd0f,TRUE,Unknown Location,,,,,,,,,,,,,,,
-0d9bf0c2-39f5-489a-a660-927f1ac97b7b,,LOCATION_CARDIOLOGY,LOCATION_CARDIOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-c6e23237-5a0d-438a-9381-ac403d30054f,,LOCATION_DELIVERY,LOCATION_DELIVERY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-79157d5d-1e45-407d-bf15-ae8dc1674a70,,LOCATION_DENTISTRY,LOCATION_DENTISTRY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-8568c575-16fc-4330-a3bb-c0460c15a298,,LOCATION_GYNAECOLOGY,LOCATION_GYNAECOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-df204697-a6c6-4768-a4e8-7082940fc8eb,,LOCATION_OPTHALMOLOGY,LOCATION_OPTHALMOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-4dbf4014-4f2c-4c74-ad3e-2f6df3f60c5d,,LOCATION_RADIOLOGY,LOCATION_RADIOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
+
```

Comment:
@prashantsable1991 please remove the 2 empty lines, this is causing errors in the back-end log:
```
ERROR - CsvParser.saveAll(184) |2019-07-09 09:34:43,501| An OpenMRS object could not be constructed or saved from the following CSV line: [null]
ERROR - CsvParser.saveAll(184) |2019-07-09 09:34:43,504| An OpenMRS object could not be constructed or saved from the following CSV line: [null]
```